types/web-resource-inliner/index.d.ts

```diff
@@ -0,0 +1,136 @@
+         * @default true
+         */
+        links?: boolean | number;
+
+        /**
+         * Describes the path relationship between where web-resource-inliner is running and what the relative paths in fileContent or href/src urls refer to.
+         *
+         * For example, the tests cases in this package are in test/cases/ so their relative paths start by referring to that folder,
+         * but the root of this project and where npm test runs from is 2 folders up, so relativeTo is set to test/cases/ in test/spec.js.
+         * Likewise, with href="content.css" and a relativeTo of http://github.com/ the resource retrieved would be http://github.com/content.css.
+         *
+         * @default ""
+         */
+        relativeTo?: string;
+        /**
+         * Describes the path relationship between CSS content and the context it will be loaded in.
+         *
+         * For example, when a CSS file contains url(some-file.png) and the file is moved from a location in a folder like /css to /
+         * then the path to the image needs to be changed to url(css/some-file.png).
+         * In this case, rebaseRelativeTo would be css. You probably don't want to set this when you are using html().
+         *
+         * @default ""
+         */
+        rebaseRelativeTo?: string;
+
+        /**
+         * When strict is true, a missing resource will cause the inliner to halt and return an error in the callback.
+         *
+         * The default behavior is to log a warning to the console and continue inlining with the available resources, which is more similar to how a web page behaves.
+         *
+         * @default false
+         */
+        strict?: boolean;
+
+        /**
+         * Allows to adjust issued requests.
+         *
+         * E.g., add authentication tokens to requested URLs.
+         *
+         * The function is called with the request options object as its parameter.
+         *
+         * It can modify this object or return a new one.
+         *
+         * See the [list of available options](https://www.npmjs.com/package/request#request-options-callback).
+         */
+        requestTransform?: (request: object) => object;
+        /**
+         * Allows to make changes to scripts before they are inlined, such as minifying.
+         *
+         * Callback is standard node error first, second argument is transformed value.
+         */
+        scriptTransform?: (content: string, callback: Callback) => any;
+        /**
+         * Allows to make changes to links before they are inlined, such as CSS pre-and-post-processors.
+         *
+         * Callback is standard node error first, second argument is transformed value.
+         */
+        linkTransform?: (content: string, callback: Callback) => any;
+    }
+
+    function html(options: Options, callback: Callback): string;
+    function css(options: Options, callback: Callback): string;
+}
+
+declare function WebResourceInliner(): void;
```

Comment:
It appears that the top level of this package is not actually a function `WebResourceInliner`, but rather an object containing the `html` and `css` functions. If that's true, you should use [this template](http://www.typescriptlang.org/docs/handbook/declaration-files/templates/module-d-ts.html) rather than the namespace-merging structure you currently have.

Reply:
I don't understand what i did wrong